Add Flathub button and some Linux-specific info to README.me (#301)
upscayl/upscayl: README.md

  </br>
  
<a href="https://github.com/upscayl/upscayl/releases/latest">
  <img src="https://github.com/upscayl/upscayl/assets/25067102/6287fd40-2c91-4028-b1d6-3986e77d8211" width="200px" />
</a>

<a href="https://t.me/iamnayam">
  <img src="https://user-images.githubusercontent.com/25067102/209297095-a3db856f-b760-40bb-a68e-f3a3086e18c7.png" width="200px" />   
</a>      

<a href="https://t.me/LinuxChats">
  <img src="https://user-images.githubusercontent.com/25067102/215301815-b9922b27-4e65-435a-aa9d-239da0ac61b4.png" width="200px" />
</a>

# 🆙 Upscayl 

#### Free and Open Source AI Image Upscaler

> 

Reviewer: ```suggestion
Upscayl lets you enlarge and enhance low-resolution images using advanced AI algorithms. It's almost like magic! 🎩🪄
```

Reviewer 2: I think this should be committed to master instead of relying onthis PR

Reviewer: Hmm, it's just one line change and I didn't want to cause any merge conflicts, so I thought it'd be better to just suggest it here.

  </a>

  <a href='https://flathub.org/apps/org.upscayl.Upscayl'>
    <img src='https://dl.flathub.org/assets/badges/flathub-badge-en.png' alt='Download on Flathub' Height="70px" maxWidth='250px'/>
  </a>

  <a href="https://aur.archlinux.org/packages/upscayl-bin">
    <img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" width="250px"/>
  </a>
</div>

#### [Flatpak](https://flatpak.org)

1. Go to [Flathub/Upscayl](https://flathub.org/apps/org.upscayl.Upscayl)

2. Press the "Install" button. This will download the `.flatpakref` file.

3. Execute the downloaded `.flatpakref` file. This will open your disrtibution's software center.

4. Press the installation button in your disrtibution's software center.

#### [AppImage](https://appimage.org/)

1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)

2. Download the `.AppImage` file.

3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.

*Please note:*
- *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*

> - *The AppImage needs to be [made executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80) after download.*

Reviewer 2: Should these notes be in the individual sections instead? Like the first one under flatpak and the second one under AppImage? Also, I don't think we should link the titles, as these are all notable enough and no other heading is linked.

Author: [Done](https://github.com/Danik1601/upscayl/commit/c38162f86f66a43c067a30b72a1a1ade57888391)

  
</div>

# 👨‍💻 Installation 

### 🐧 Linux 

  <a href="https://flathub.org/apps/org.upscayl.Upscayl">
    <img src="https://dl.flathub.org/assets/badges/flathub-badge-en.svg" height="50px"/>
  </a>

  <a href="https://appimage.github.io/Upscayl/">
    <img src="https://user-images.githubusercontent.com/25067102/191270389-9de37c0f-39a8-41f1-a659-8dd4e7b8ac28.png" height="50px"/>
  </a>

  <a href="https://aur.archlinux.org/packages/upscayl-bin">
    <img src="https://user-images.githubusercontent.com/25067102/191269445-87050a77-c304-4284-9ea0-699721309c59.png" height="50px"/>
  </a>

  <a href="https://snapcraft.io/upscayl/">
    <img src="https://snapcraft.io/static/images/badges/en/snap-store-black.svg" height="50px"/>
  </a>

> #### Flatpak
> 
> 1. Go to your distribtuion's software center (or your favourite installed one) and search for Upscayl.  
> Alternatively you can go to [Flathub/Upscayl](https://flathub.org/apps/org.upscayl.Upscayl) and click "Install".
> 
> 2. Download Upscayl using your distribution's software center
> 
> *Please note:*
> - *On some distributions Flatpak must be [pre-configured manually](https://flatpak.org/setup).*
> 
> #### AppImage
> 
> 1. Go to [releases section](https://github.com/TGS963/upscayl/releases/latest)
> 
> 2. Download the `.AppImage` file.
> 
> 3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.
> 

Reviewer: ```suggestion
Upscayl should be available on the Software Store of most Linux operating systems. You can download the Snap or [Flatpak](https://flatpak.org/setup) version, if your distro's Store App supports them.

#### Manual Install

1. Go to [releases section](https://github.com/upscayl/upscayl/releases/latest)

2. Download the `upscayl-x.x.x-linux.AppImage` file.

3. Right Click AppImage -> Go to Permissions tab -> Check allow file to execute and then double click the file to run Upscayl.

You can also choose to install using other formats like RPM (Fedora), DEB (Debian/Ubuntu based), and ZIP (Any Linux OS).
```

Reviewer 2: I don't think an AppImage is a "Manual Install", which sounds like compiling it.

Reviewer: > I don't think an AppImage is a "Manual Install", which sounds like compiling it.

Good point

> *Please note:*
> - *The AppImage needs to be [made executable](http://discourse.appimage.org/t/how-to-make-an-appimage-executable/80) after download.*

Reviewer: ```suggestion
```

This isn't needed because the instructions already mention how to make it an executable.

# Donate

<a href="https://www.buymeacoffee.com/fossisthefuture">
<img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
  <img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
<a/>

# Credits ❤

- Real-ESRGAN for their wonderful research work.\
- Real-ESRGAN for their wonderful research work\.  
[Real-ESRGAN: Copyright (c) 2021, Xintao Wang](https://github.com/xinntao/Real-ESRGAN/)

> 

Reviewer 2: Is the backslash even needed now that the extra newline is gone?

Author: No it is not, you are right. Double space will do the job perfectly. [Done](https://github.com/Danik1601/upscayl/commit/b6b93377f2ea5d32ac3020e5fb6cbb4bfe683647)

Author: By the way, can we just insert the link into `Real-ESRGAN` and remove the whole `Real-ESRGAN: Copyright (c) 2021, Xintao Wang` line altogether?

Reviewer 2: Maybe. You'll have to ask @NayamAmarshe though.

Reviewer: I think we should give proper credits, the current one is fine :)
